leveldb/leveldb.go

```diff
@@ -0,0 +1,216 @@
+package leveldb
```

Comment:
todo: Please document the limitations noted in the PR description here, they are very important to know for anyone consuming this package.